src/util/Object.js

```diff
@@ -55,6 +55,34 @@ const ObjectUtil = {
       result.push(path);
     }
     return result;
+  },
+
+  /**
+   * Returns the value from an object using a case-insensitive key.
+   *
+   * @param {Object} obj The object to search in
+   * @param {*} key The key to retrieve value for
```

Comment:
```suggestion
   * @param {String} key The key to retrieve value for
```

Shouldn't key be declared as String so IDEs can report wrong usage ?